sample/app/src/main/java/app/cash/paraphrase/sample/app/MainActivity.kt

```diff
@@ -83,70 +87,74 @@ class MainActivity : ComponentActivity() {
   companion object {
     data class Sample(val label: String, val resource: FormattedResource)
 
-    private val APP_SAMPLES = listOf(
+    private fun appSamples(
+      formattedResources: AppFormattedResources,
+    ): List<Sample> = listOf(
```

Comment:
If we wanted to keep this as a `val` with static references to `AppAndroidFormattedResources` that would still work right? Are we just switching to a `fun` to demonstrate that you can pass the resources as the interface type?